### PR TITLE
fix: change more view position in monthly view (fix #300)

### LINKED ISSUE #300: In a monthly view, the More View of last row and column is cropped
## Version

v1.11.0

## Development Environment

Mac OS Mojave, Chrome 73

## Current Behavior

In a monthly view, the More View of last row and column is cropped
스크린샷 2019-04-09 16 21 42
스크린샷 2019-04-09 16 21 58
스크린샷 2019-04-09 16 21 50

## Expected Behavior

cal-month-view-last-col
cal-month-view-last-row
cal-month-view-last-row-n-col

### PR DESCRIPTION
**Pull request by jungeun-cho**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has description for the breaking change

### Description
* Modify that the More View of the last row and column is displayed correctly in a monthly view.
cal-month-view-last-row
cal-month-view-last-col
cal-month-view-last-row-n-col

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: examples/js/app.js [hunk 1]
```diff
@@ -12,7 +12,7 @@
     var datePicker, selectedCalendar;
 
     cal = new Calendar('#calendar', {
-        defaultView: 'week',
+        defaultView: 'month',
```
Comment (dongsik-yoo) [inline examples/js/app.js]: app.js 파일은 바꾸면 캘린더 타입이 바뀌어서 원래대로 주간뷰로 해주세요~

File: src/js/view/month/more.js [hunk 2]
```diff
@@ -144,6 +144,9 @@ More.prototype.render = function(viewModel) {
     var styles = this._getStyles(this.theme);
     var maxVisibleSchedulesInLayer = 10;
     var height = '';
+    var isLastRow = weekItem.parentElement.lastElementChild === weekItem;
+    var isLastCol = target.parentElement.lastElementChild === target;
+    var minWidth = 280;
```
Comment (dongsik-yoo) [inline src/js/view/month/more.js]: 매직넘버라 변수로 빼는게 좋겠어요.

File: examples/js/app.js (hunk 1 above)
Comment (jungeun-cho) [reply examples/js/app.js]: 네네 수정하였습니다.(예제파일원복)

File: src/js/view/month/more.js (hunk 2 above)
Comment (jungeun-cho) [reply src/js/view/month/more.js]: VEIW_MIN_WIDTH로 수정합니다.

**Pull request merged**